Formula/fish.rb

```diff
@@ -39,7 +54,30 @@ def caveats; <<-EOS.undent
     EOS
   end
 
+  def post_install
+    (share/"fish/vendor_functions.d").mkpath
```

Comment:
These are what the audit is complaining about; they can be `pkgshare/"vendor_functions.d"` etc.